doc/api/deprecations.md

```diff
@@ -1058,14 +1075,18 @@ changes:
     description: Documentation-only deprecation.
 -->
 
-Type: Documentation-only
+Type: Runtime
 
-The [`util.isError()`][] API is deprecated.
+The [`util.isError()`][] API is deprecated. Please use
+`arg instanceof Error` instead.
```

Comment:
this one, in particular, is *impossible* to do in userland, so it's pretty important i think to keep it.

Reply:
We should remove doc deprecation as well if something is considered valuable and has no good replacements

Reply:
`function isError(e) {
  return ObjectPrototypeToString(e) === '[object Error]' || e instanceof Error;
}`
This what the `isError` function is doing

Reply:
then that should be the migration recommendation

Reply:
@marco-ippolito ah, i thought it was checking an internal slot like isProxy - if that's what it's doing then it's pretty useless. The rec still shouldn't solely rely on instanceof tho.

Reply:
@ljharb it should be something weird like:
`function isError(e) { return Object.prototype.toString(e) === '[object Error]' || e instanceof Error; }`

Reply:
Right - that's more or less the best check you can do, because it works across realms.